templates/try-matrix.html

```diff
@@ -44,7 +49,11 @@ <h3>Register and start chatting</h3>
                     browse the <a href="/docs/chat_basics">chat basics</a>
                     documentation.
                 </p>
-                <a href="/docs/chat_basics" class="call-to-action">Read the docs</a>
+                <div class="cta-wrapper">
+                    <a href="/docs/chat_basics/matrix-for-im/#creating-a-matrix-account" class="call-to-action">Create
+                        an account</a>
+                    <a href="/docs/chat_basics" class="call-to-action secondary">Read the docs</a>
```

Comment:
More wording nitpicking, sorry. The Register and start chatting section has a "Start chatting" button. This should IMHO lead to https://app.element.io/#/register rather than just to some longish manual. Also the "Read the docs" is a very generic term that *I* did not know what to expect (the spec?). Perhaps call it "Read client manual" or something. I'll stop here.